pkg/kubectl/cmd/get/get.go

```diff
@@ -420,6 +418,13 @@ func (o *GetOptions) Run(f cmdutil.Factory, cmd *cobra.Command, args []string) e
 			lastMapping = mapping
 		}
 
+		// ensure a versioned object is passed to the custom-columns printer
```

Comment:
@soltysh ptal at this

Reply:
That's not needed, this should work nicely with the next ifs, @juanvallejo any particular reason for doing this?